CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Note: This is in reverse chronological order, so newer entries are added to the
 Swift 5.8
 -----------
 
+* [#5966]
+
+  Plugin compilation can be influenced by using `-Xhost` arguments in the `swift build` command line. This is similar to the existing mechanism for influencing the manifest compilation using `-Xmanifest` arguments. Both `-Xhost` and `-Xmanifest` are aliases of each other and will influence both the manifest and plugin compilation.
```

Comment:
Could you replace `swift build` with SwiftPM here? These flags are accepted by multiple tools, e.g. `swift test`